cartridge/semver.lua

```diff
@@ -0,0 +1,154 @@
+--
```

Comment:
Why don't you use https://github.com/tarantool/vshard/blob/master/vshard/version.lua directly?

Reply:
unfortunately vshard is not released yet and cartridge have to base on last stable vshard release
semver.lua will be definitely replaced in future